hphp/util/asm-x64.h

```diff
@@ -893,141 +627,125 @@ struct X64Assembler {
 #undef BYTE_LOAD_OP
 #undef BYTE_STORE_OP
 #undef BYTE_REG_OP
+#undef IMM64_STORE_OP
+#undef IMM64R_OP
```

Comment:
Drop this line, it's already undef in 622 above.

Reply:
Addressed in https://github.com/facebook/hhvm/pull/8164/commits/ac4a93085f536b08c1f366528bb51a4599d8b290